### PR TITLE
Move EmojiConverter to service

### PR DESCRIPTION
**Pull request by atsmith813**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
This PR moves and updates the `EmojiConverter` from `/labor` into `/services`.

## Related Tickets & Documents
https://github.com/forem/InternalProjectPlanning/issues/271

## Added tests?
- [x] Yes

## Added to documentation?
- [x] No documentation needed

dog_driving_gif

### CODE REVIEW

**Review by maestromac — APPROVED** [review]

File: app/services/emoji_converter.rb [hunk 1]
```diff
@@ -1,15 +1,7 @@
 class EmojiConverter
```
Comment (rhymes) [inline app/services/emoji_converter.rb:1]: I think we should change the name of the service to an action, see https://docs.dev.to/backend/service-objects/

Something like `Html::ParseEmoji`, this way we can also move `CodeBlockParser` to something lik `Html::ParseCodeBlock` 

What do you think?

**Review by rhymes — CHANGES_REQUESTED** [review]
Left a request change for naming

File: app/services/emoji_converter.rb (hunk 1 above)
Comment (atsmith813) [reply app/services/emoji_converter.rb]: Great idea - updated!

**Review by vaidehijoshi — APPROVED** [review]

**Review by rhymes — APPROVED** [review]
LGTM!

Thanks for the change @atsmith813

**Pull request merged**